### ISSUE TITLE
Update support for the latest Leap/Oculus SDKs

### ISSUE THREAD

**Issue by zachkinstner** [aestheticinteractive/Hover-VR-Interface-Kit#32]
[This forum post](https://community.leapmotion.com/t/unity-assets-ar-vr-update/3269) has all the relevant information for compatibility between the Leap SDK, the Oculus SDK, and Unity. Rather than provide an update for the various intermediate steps, I will probably wait until:

> [Leap Motion] Front End team will soon start work on the new Core Assets for **Unity 5.1.3** and the upcoming **Oculus 0.7 SDK**. We’re targeting them to land **before [...] Sept. 28th**.

**Comment by ilanb** [comment]
Hi zachkinstner,

Can you tell me what version of unity, oculus and leapmotion should I use to play with your 1.3 version of Hover-VR-Interface-Kit.
I'm on Mac...

Thanks

**Comment by zachkinstner** [comment]
Hi @ilanb, you should be able to get the following combination to work:

**Oculus 0.5:** Newer versions are not compatible with Mac, see [this question](https://forums.oculus.com/viewtopic.php?t=24403)

**Leap Motion 2.2.4:** Because [version 2.2.6](https://developer.leapmotion.com/documentation/Leap_SDK_Release_Notes.html#version-2-2-6) changed `Finger.Type` from method to property, the 2.2.6+ versions are not immediately compatible with Hover VR. You _can_ update (and rebuild) your local Hover VR code to be compatible with the latest Leap Motion version, see my instructions in [this forum discussion](https://community.leapmotion.com/t/hover-vr-interface-integration-errors/3567).

**Unity 4.6 or 5.0:** The demos within this project all use Unity 4.6, but lots of us have been using Hover VR in Unity 5 projects. You may need to let Unity do it's auto-upgrade process when installing the Hover VR Unity package. There is a known issue when using Unity 5.1+, see my workaround instructions in [this forum discussion](https://community.leapmotion.com/t/hover-vr-interface-integration-errors/3567).

**Comment by ilanb** [comment]
Thanks for quick reply zachkinstner !

**Comment by zachkinstner** [comment]
The new Leap Motion assets (Core Assets 3.0.0) are now available. I have started work on this upgrade today -- see my [commits](https://github.com/aestheticinteractive/Hover-VR-Interface-Kit/commits/master) on the `master` branch from Sep 28, 2015.

This upgrade will require some rework of the existing demo scenes, since the VR-based cameras are now supported directly by Unity (rather than placing an Oculus prefab into the scene). I did convert one demo scene so far, and it seems to work as expected.
